packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -154,6 +157,27 @@ if (__DEV__) {
   };
 }
 
+export function logError(boundaryFiber: Fiber, capturedError: CapturedError) {
+  const errorBoundaryFound = boundaryFiber.tag === ClassComponent;
+  capturedError.errorBoundary = boundaryFiber;
+  capturedError.errorBoundaryFound = errorBoundaryFound;
+  capturedError.errorBoundaryName = errorBoundaryFound
+    ? getComponentName(boundaryFiber)
+    : null;
+  // TODO: These are always the same. Why is it needed?
+  capturedError.willRetry = errorBoundaryFound;
+  try {
+    logCapturedError(capturedError);
+  } catch (e) {
+    // Prevent cycle if logCapturedError() throws.
+    // A cycle may still occur if logCapturedError renders a component that throws.
+    const suppressLogging = e && e.suppressReactErrorLogging;
+    if (!suppressLogging) {
+      console.error(e);
+    }
```

Comment:
Unrelated to this PR but I wonder what happens if `console.error` throws. Can happen if somebody overrides it for blacklisting and messes up. I hope we don’t go into an infinite loop.